app/views/tracks.js

```diff
@@ -11,6 +11,109 @@ const TracksView = Mn.CompositeView.extend({
 
     childView: TrackView,
 
+    events: {
+         'mousedown .drag': 'dragStart'
+    },
+
+    initialize() {
+        this.dragObj = {
+            'dragDown': false,
+            'model': undefined,
+            'newIndex': undefined
+        };
+        this.listenTo(this.collection, 'sync', this.bindDragEvents);
+    },
+
+    bindDragEvents() {
```

Comment:
You bind events, but never unbind them, so you will add listener everytime a "sync" is triggered on the collection or underlying model. 
You should unbind 
- on TracksView::remove
- if bindDragEvents() is called more than once.